packages/evolution-common/src/config/project.config.ts

```diff
@@ -25,6 +26,24 @@ export type EvolutionProjectConfiguration = {
     /** Used for Google Maps localization. See
      * https://developers.google.com/maps/coverage for possible region codes */
     region: string;
+    /**
+     * Start date time for the survey in ISO date time string format with timezone offset
+     * (YYYY-MM-DDTHH:MM:SS-/+HH:MM). Example: 2025-01-01T00:00:00-05:00.
+     * Interviews started before this date and time should be invalidated and/or ignored.
+     * If both startDateTimeWithTimezoneOffset and endDateTimeWithTimezoneOffset are defined,
+     * endDateTimeWithTimezoneOffset must be after startDateTimeWithTimezoneOffset.
+     * Provide the timezone offset so we can calculate the correct unix epoch.
```

Comment:
What if it is not provided. Say someone just enters '2025-01-01' because they don't care, will it still work? Would any string convertible to datetime format work?

Reply:
On doit vérifier ça dans le survey... Comment tester ça dans evolution?